project/Build.scala

```diff
@@ -36,6 +36,7 @@ object B extends Build {
         case "2.12.0-M2" => "2.2.5-M2"
         case "2.12.0-M3" => "2.2.5-M3"
         case "2.12.0-M4" => "2.2.6"
+        case "2.12.0-M5" => "3.0.0-RC4"
```

Comment:
No, @bvenners only released 3.0.0 for 2.12.M5 so far.
Perhaps we could move on as well? Source code wise it should not be that painful (here in this repo no changes were required)